libs/cli/setup.go

```diff
@@ -91,7 +91,7 @@ func (e Executor) Execute() error {
 		if viper.GetBool(TraceFlag) {
 			fmt.Fprintf(os.Stderr, "ERROR: %+v\n", err)
 		} else {
-			fmt.Fprintf(os.Stderr, "ERROR: %v\n", err)
+			fmt.Fprintf(os.Stderr, "ERROR: %v\n* Have you forgotten to execute tendermint init?\n", err)
```

Comment:
won't this be run on any CLI error? I don't think that is what we want.

Reply:
You're right. I'm still pretty new to blockchain stuff as a whole so diving into this repo has been a bit daunting. I'll hopefully have this resolved in the coming days as I get my bearings a bit better.